packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -194,6 +194,10 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
       (this.props as any)['data-is-focusable'] === false || (disabled && !allowDisabledFocus) || this._isSplitButton ? false : true;
 
     const isCheckboxTypeRole = role === 'menuitemcheckbox' || role === 'checkbox';
+    // if isCheckboxTypeRole, always return a checked value, otherwise only return checked value if toggle is set to true
+    // This is because role="checkbox" always needs to have an aria-checked value
+    // but our checked prop only sets aria-pressed if we mark the button as a toggle="true"
+    const checkedOrPressedValue = isCheckboxTypeRole ? !!checked : toggle === true ? !!checked : undefined;
```

Comment:
Do we really need the `toggle === true` check? Does `toggle` alone not work?

Reply:
I did that to make it obvious it was a condition and not a value. sometimes those nested ternaries can be annoying to read.

Reply:
```jsx
const checkedOrPressedValue = isCheckboxTypeRole ? !!checked : toggle ? !!checked : undefined;
```
if isCheckboxTypeRole is false then return toggle? no..., first check for toggle===true then return !!checked or undefined